cdap-ui/app/cdap/components/LoadingIndicator/index.js

```diff
@@ -40,7 +40,10 @@ export default class LoadingIndicator extends Component {
 
   componentDidMount() {
     this.loadingIndicatorStoreSubscription = LoadingIndicatorStore.subscribe(() => {
-      if (location.pathname.indexOf('/cdap/administration') !== -1) {
+      if (
+        location.pathname.indexOf('/cdap/administration') !== -1 ||
```

Comment:
nit: we may also use `includes` instead of `indexOf`

Reply:
Done!